examples/proteus.linux2.yaml

```diff
@@ -37,7 +37,8 @@ packages:
   coverage:
   hdf5:
   petsc:
-    version: 3.8.4
+    version: 3.7.6
```

Comment:
why is the version changed in the linux2 file? (copper uses the copper.gnu file)